requirements/dev-requirements.txt

```diff
@@ -548,10 +548,8 @@ pyjwt[crypto]==2.7.0
     #   twilio
 pynacl==1.5.0
     # via pygithub
-pyopenssl==23.2.0
-    # via
-    #   -r docs-requirements.in
-    #   -r sso-requirements.in
+pyopenssl==24.0.0
```

Comment:
Yes, it's required for `docs-requirements` and because of that, it's also included in `dev-requirements` because `docs-requirements` is included in `dev-requirements`.

You are correct, my PR title was not quite accurate since we still depend on it for our docs builds.